src/frontend/classic/components/MediaSelect.svelte

```diff
@@ -49,13 +50,16 @@
 
     let mediaNameFilter = "";
     $: {
```

Comment:
Maybe remove brackets for surounding reactive block, as the `if` may already count as a block

Reply:
Thank you for your kind suggestion, the proposed change has been applied at https://github.com/manga-download/haruneko/pull/37/commits/71a38ab46e4e33134c5ccf8766df3b3d90bfe9d1